srml/support/test/tests/reserved_keyword/on_initialize.stderr

```diff
@@ -1,47 +1,47 @@
 error: Invalid call fn name: `on_finalize`, name is reserved and doesn't match expected signature, please refer to `decl_module!` documentation to see the appropriate usage, or rename it to an unreserved keyword.
   --> $DIR/on_initialize.rs:39:1
    |
-30 | reserved!(on_finalize on_initialize on_finalise on_initialise offchain_worker deposit_event);
+39 | reserved!(on_finalize on_initialize on_finalise on_initialise offchain_worker deposit_event);
```

Comment:
Ah this is from doughnut changes, thought I had fixed this 😅

Reply:
Yep I got ur back on this one 😃